modular_chomp/code/modules/mob/new_player/sprite_accessories_extra.dm

```diff
@@ -483,3 +483,268 @@
+	icon_state = "thin2"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_R_FOOT,BP_R_LEG)
+	body_parts = list(BP_R_FOOT,BP_R_LEG)
+
+// THE SKELLYWAG //
+
+/datum/sprite_accessory/marking/ch/skeleton_full
+	name = "Skeleton (Neck Down)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG,BP_L_ARM,BP_R_ARM,BP_L_HAND,BP_R_HAND,BP_GROIN,BP_TORSO)
+	body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG,BP_L_ARM,BP_R_ARM,BP_L_HAND,BP_R_HAND,BP_GROIN,BP_TORSO)
+
+/datum/sprite_accessory/marking/ch/skeleton_full_nohide
+	name = "Skeleton (Neck Down, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG,BP_L_ARM,BP_R_ARM,BP_L_HAND,BP_R_HAND,BP_GROIN,BP_TORSO)
+
+/datum/sprite_accessory/marking/ch/skeleton_torso
+	name = "Skeleton (Torso)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_GROIN,BP_TORSO)
+	body_parts = list(BP_GROIN,BP_TORSO)
+
+/datum/sprite_accessory/marking/ch/skeleton_leftarm
+	name = "Skeleton (Left Arm)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_L_ARM,BP_L_HAND)
+	body_parts = list(BP_L_ARM,BP_L_HAND)
+
+/datum/sprite_accessory/marking/ch/skeleton_rightarm
+	name = "Skeleton (Right Arm)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_R_ARM,BP_R_HAND)
+	body_parts = list(BP_R_ARM,BP_R_HAND)
+
+/datum/sprite_accessory/marking/ch/skeleton_leftleg
+	name = "Skeleton (Left Leg)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_L_FOOT,BP_L_LEG)
+	body_parts = list(BP_L_FOOT,BP_L_LEG)
+
+/datum/sprite_accessory/marking/ch/skeleton_rightleg
+	name = "Skeleton (Right Leg)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	hide_body_parts = list(BP_R_FOOT,BP_R_LEG)
+	body_parts = list(BP_R_FOOT,BP_R_LEG)
+
+/datum/sprite_accessory/marking/ch/skeleton_torso_nohide
+	name = "Skeleton (Torso, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_GROIN,BP_TORSO)
+
+/datum/sprite_accessory/marking/ch/skeleton_leftarm_nohide
+	name = "Skeleton (Left Arm, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_L_ARM,BP_L_HAND)
+
+/datum/sprite_accessory/marking/ch/skeleton_rightarm_nohide
+	name = "Skeleton (Right Arm, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_R_ARM,BP_R_HAND)
+
+/datum/sprite_accessory/marking/ch/skeleton_leftleg_nohide
+	name = "Skeleton (Left Leg, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_L_FOOT,BP_L_LEG)
+
+/datum/sprite_accessory/marking/ch/skeleton_rightleg_nohide
+	name = "Skeleton (Right Leg, No Hide)"
+	icon_state = "skeleton"
+	color_blend_mode = ICON_MULTIPLY
+	body_parts = list(BP_R_FOOT,BP_R_LEG)
```

Comment:
These duplicates should be deleted.

Marking bodyparts can now be toggled per-limb now (and recolored per limb) meaning there's less need for 'marking l arm' 'marking r arm' and just a singular 'making full body' like the skeleton_full and skeleton_full_nohide.

Doing this will cut down on marking bloat and savefile migrations later down the line.